### PR TITLE
Video tracks

### PR DESCRIPTION
**Pull request by kubaau**

### CODE REVIEW

File: DolbyIO/DolbyIO.uplugin [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"FileVersion": 3,
 	"Version": 1,
-	"VersionName": "1.1.0-beta.1",
+	"VersionName": "2.0.0-beta.1",
```
Comment (kubaau) [inline DolbyIO/DolbyIO.uplugin:4]: This should be 1.1.0-beta.2 probably.

File: DolbyIO/Source/Private/DolbyIOVideoSink.cpp [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 namespace DolbyIO
 {
-	UTexture2D* FVideoSink::GetTexture()
+	UTexture2D* FVideoSink::GetTexture() const
```
[the remark this answers is not in the document]
Comment (kubaau) [reply DolbyIO/Source/Private/DolbyIOVideoSink.cpp:19]: Could you rephrase that? I don't see anything wrong.

File: DolbyIO/Source/Private/DolbyIOVideoFrameHandler.h [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+#pragma once
+
+#include "DolbyIOCppSdk.h"
+
+namespace DolbyIO
+{
+	class FVideoFrameHandler final : public dolbyio::comms::video_frame_handler
+	{
+	public:
+		FVideoFrameHandler(std::shared_ptr<dolbyio::comms::video_sink> Sink);
```
[the remark this answers is not in the document]
Comment (kubaau) [reply DolbyIO/Source/Private/DolbyIOVideoFrameHandler.h:12]: Could you rephrase that? Where would you like it to be defined instead?

[the remark this answers is not in the document]
Comment (Adam1-3) [reply DolbyIO/Source/Private/DolbyIOVideoFrameHandler.h:12]: The constructor is declared in .h file ("here") and defined in .cpp while other methods are defined "here" - The proposal is to define all methods in .h file and remove .cpp file. A very low priority tip.

**Review by Adam1-3 — APPROVED** [review]
